### DISCUSSION TITLE
devDependencies or dependencies?

### DISCUSSION THREAD

**Discussion by stephenjason89** [nuxt/nuxt#24963] [General]
I noticed the latest stackblitz https://stackblitz.com/github/nuxt/starter/tree/v3/?file=package.json 
<img width="373" alt="image" src="https://github.com/nuxt/nuxt/assets/3810177/f213a0b1-1015-457b-b758-51960bcb55f4">

has nuxt and vue as depDependencies, however, latest patch notes placed it in dependencies https://github.com/nuxt/nuxt/releases/tag/v3.9.0
<img width="1190" alt="image" src="https://github.com/nuxt/nuxt/assets/3810177/54bdcb44-d4e8-46a5-91a7-df9c4f6063cc">

But, doing `nuxi upgrade --force` automatically moves nuxt dependencies to devDependencies.

I'm a little confused, if anyone can shed some light here.

Thank you

**Comment by larrydevofficial** [comment]
I would like an explanation of this behavior as well, performing nuxi upgrade --force does indeed move nuxt package into devDependencies I now have a bug in a server route and I expect this behavior could be why.

**Comment by unreadableusername** [comment]
Did you figure it out? 
I am in the same situation with Nuxt 3.11 refusing to update to 3.12 because the nuxt init script placed all dependencies in the "dependencies" field and not "devDependencies".

**Comment by meirroth** [comment]
@stephenjason89, @larrydevofficial and @unreadableusername See [Atinux](https://github.com/Atinux)'s response to this question here: https://github.com/nuxt/starter/issues/723#issuecomment-2124842101
